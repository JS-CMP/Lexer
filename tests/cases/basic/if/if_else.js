if (false) {
    console.log("False block")
} else {
    console.log("Else block should run")
}
