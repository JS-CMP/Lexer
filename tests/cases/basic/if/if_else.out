#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {if (JS::Any(false)){console[u"log"](JS::Any(u"False block"));}else {console[u"log"](JS::Any(u"Else block should run"));};}
